applications/DEMApplication/CMakeLists.txt~HEAD

```diff
@@ -0,0 +1,146 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
Please clean files that don't belong here. Ask @AlejandroCornejo , once you are done I will review

Reply:
yeah, we ar working on it